target/linux/ramips/image/mt7621.mk

```diff
@@ -286,6 +286,25 @@ define Device/arcadyan_we420223-99
 endef
 TARGET_DEVICES += arcadyan_we420223-99
 
+define Device/zte_mf286c3
+  $(Device/nand)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  VID_HDR_OFFSET := 2048
+  IMAGE_SIZE := 41984k
+  KERNEL_SIZE := 5120k
+  UBINIZE_OPTS := -E 5
+  DEVICE_VENDOR := ZTE
+  DEVICE_MODEL := MF286C3
+  DEVICE_PACKAGES := kmod-mt7603 kmod-sdhci-mt7620 kmod-usb2 kmod-usb3 kmod-fs-nfs-v3 kmod-fs-nfs-v4 kmod-l2tp kmod-mt7615e \
+  kmod-mt7663-firmware-ap kmod-mt7663-firmware-sta -wpad-basic-mbedtls -uboot-envtools \
+  kmod-usb-ledtrig-usbport nand-utils kmod-i2c-core hostapd \
+  kmod-usb-net kmod-usb-wdm kmod-usb-net-qmi-wwan kmod-usb-serial-qualcomm kmod-usb-serial-option \
+  kmod-usb-net-rndis
```

Comment:
uqmi and rndis - are both required?
and serial-qualcomm and serial-option - which one?

Reply:
removed unnecessary things, all that is additionally needed will be added to the image builder